modules/bootstrap-ks/Makefile

```diff
@@ -0,0 +1,353 @@
+#-- bootstrap-ks Params---#
+BOOTSTRAP_KS_REPO ?= https://${GITHUB_TOKEN}@github.com/open-cluster-management/bootstrap-ks.git
+BOOTSTRAP_KS_BRANCH ?= master
+BOOTSTRAP_KS_DEPLOY_DIR ?= bootstrap-ks
+
+#---Login/Host-cluster related env vars---#
+# API URL of the cluster running the target clusterpool, used in oc login, so port :6443 is required
+BOOTSTRAP_KS_HOST_API ?=
+# Access credentials used to log in to cluster indciated in BOOTSTRAP_KS_HOST_API
+BOOTSTRAP_KS_HOST_USER ?=
+BOOTSTRAP_KS_HOST_PASS ?=
+BOOTSTRAP_KS_HOST_TOKEN ?=
+# Namespace where Hive ClusterPool resources reside
+BOOTSTRAP_KS_HOST_NAMESPACE ?= clusterpool
+
+OC_CLUSTER_USER = BOOTSTRAP_KS_HOST_USER
+OC_CLUSTER_PASS = BOOTSTRAP_KS_HOST_PASS
+OC_CLUSTER_API = BOOTSTRAP_KS_HOST_API
+
+CLUSTER_NAME_PREFIX ?= canary
+
+# These will only have a value if the cluster has been deployed.  (Only one per type is allowed)
+BOOTSTRAP_KS_AKS_CLUSTER_NAME = $(shell cd ${BOOTSTRAP_KS_DEPLOY_DIR}/aks; cat *.json | jq -r .CLUSTER_NAME)
+BOOTSTRAP_KS_GKE_CLUSTER_NAME = $(shell cd ${BOOTSTRAP_KS_DEPLOY_DIR}/gke; cat *.json | jq -r .CLUSTER_NAME)
+BOOTSTRAP_KS_EKS_CLUSTER_NAME = $(shell cd ${BOOTSTRAP_KS_DEPLOY_DIR}/eks; cat *.json | jq -r .CLUSTER_NAME)
+BOOTSTRAP_KS_IKS_CLUSTER_NAME = $(shell cd ${BOOTSTRAP_KS_DEPLOY_DIR}/iks; cat *-iks.json | jq -r .CLUSTER_NAME)
+BOOTSTRAP_KS_ROKS_CLUSTER_NAME = $(shell cd ${BOOTSTRAP_KS_DEPLOY_DIR}/iks; cat *-roks.json | jq -r .CLUSTER_NAME)
+
+# Convenience to just call bare jq executable
+JQ ?= $(BUILD_HARNESS_PATH)/vendor/jq
+
+.PHONY: bootstrap-ks/_init
+# Install jq and oc cli and log in to the ClusterPool Host cluster
+bootstrap-ks/_init: %_init:
+	$(call assert-set,GITHUB_TOKEN)
+	$(call assert-set,BOOTSTRAP_KS_HOST_API)
+	$(call assert-set,BOOTSTRAP_KS_HOST_USER)
+	$(call assert-set,BOOTSTRAP_KS_HOST_PASS)
+#	$(call assert-set,BOOTSTRAP_KS_HOST_TOKEN)
+	@$(SELF) -s jq/install 2>&1 > /dev/null
+	@if [ ! -e "$(BOOTSTRAP_KS_DEPLOY_DIR)" ]; then \
+		git clone --single-branch --branch $(BOOTSTRAP_KS_BRANCH) $(BOOTSTRAP_KS_REPO); \
+		cd $(BOOTSTRAP_KS_DEPLOY_DIR); fi
+	@$(SELF) -s oc/login OC_CLUSTER_USER=$(BOOTSTRAP_KS_HOST_USER) OC_CLUSTER_PASS=$(BOOTSTRAP_KS_HOST_PASS) \
+		OC_CLUSTER_URL=$(BOOTSTRAP_KS_HOST_API) OC_FORCE_LOGIN="true" OC_SILENT="true"
+
+##======= Azure Kubernetes Service ===========================
+
+## Install AKS
```

Comment:
The way build harness help works is that a single line (the last line) with `##` immediately preceding the target is used for help text.  So `aks-create-cluster` is actually the only target this is currently working for.  You can elaborate on the documentation prior to the `.PHONY` tag, but the one `##` line right before the target is what gets printed during `make help`.  Internal targets (with leading underscores) don't need `##` help and can continue to use singe `#` documentation wherever appropriate.

Reply:
I was unaware of this make feature.  Will be making changes. Thanks